packages/camera-web/src/Camera/hooks/useUserMedia.ts

```diff
@@ -36,6 +36,15 @@ class InvalidStreamError extends Error {
  * @see useUserMedia
  */
 export enum UserMediaErrorType {
+  /**
+   * The camera stream couldn't be fetched because the web page does not have the permissions to access the camera.
+   * if both webpage and browser permissions are denied, this error will be returned.
+   */
+  NOT_ALLOWED_WEBPAGE = 'not_allowed_webpage',
```

Comment:
Rename error to `WEBPAGE_NOT_ALLOWED`

Reply:
@arunachalam-monk also rename the value of the variable to 'webpage_not_allowed'